.gitignore

```diff
@@ -60,4 +60,7 @@ typings/
 # next.js build output
 .next
 
-*.sqlite3
\ No newline at end of file
+*.sqlite3
+
+# ignore  any file under the server directly. to remove untracked 75 uneeded files  
+server/
```

Comment:
This rule fixes the problem of the 75 untracked files but it is risky, 
because if you have any thing else in that directory it will also be ignored !
It might sound a lot of work but you might need add the files names.